src/containers/form/FormContainer.js

```diff
@@ -101,11 +101,13 @@ class Form extends React.Component<Props, State> {
     validateOnInput(name, input, fieldType, formatErrors, setErrorsFn);
   }
 
-  handleDateInput = (e, section, name) => {
-    const input = e;
+  handleDateInput = (e, section, name, formatErrors, setErrorsFn) => {
+    let input = e;
+    input = input.replace(/T(.*)$/g, 'T00:00:00.000Z');
```

Comment:
You need to handle the case where the "clear" button was clicked, which means `input` will be `null`.

Reply:
Fixed